src/extension/app/constants.js

```diff
@@ -90,6 +89,25 @@ export const RESTRICTED_PATHS = [
   '/helix-env.json',
 ];
 
+/**
+ * Enum for sidekick states.
+ * @enum {string}
+ */
+export const SIDEKICK_STATE = {
```

Comment:
In the end, everything here is `SIDEKICK_*` 😉 
```suggestion
export const STATE = {
```

Reply:
Fair enough ;)